README.md

```diff
@@ -77,60 +77,139 @@ Notes:
 
-* seedsigner_pubkey.gpg (from the main folder of this repo)
-* seedsigner_0_4_6.img.zip (from the software release)
-* seedsigner_0_4_6.img.zip.sha256 (from the software release)
-* seedsigner_0_4_6.img.zip.sha256.sig (from the software release)
 
-**Note:** The specific version number of the files in your folder might not match the above exactly, but their overall format and amount should be the same.
+## Verifying that the downloaded files are authentic (optional but highly recommended!)
 
-This process also assumes you are running the commands from a system where both [GPG](https://gnupg.org/download/index.html) and [shasum](https://command-not-found.com/shasum) are installed and working.
+You can quickly verify that the downloaded software is both authentic and unaltered, by following these instructions.
+
+These next steps assume you are running the commands from a computer where both [GPG](https://gnupg.org/download/index.html) and [shasum](https://command-not-found.com/shasum) are already installed, and that you also know [how to navigate on a terminal](https://terminalcheatsheet.com/guides/navigate-terminal). 
+
+
+### 1. Import the public key of the SeedSigner project into your computer
+
+The *fetch-keys*  command below will import the SeedSigner projects public key from a popular online keyserver called *Keybase.io*, into your computers *keychain*. 
+
+To begin, you will run the *fetch-keys* command as shown below (run it from inside the *same folder* that you saved the downloaded files into). 
 
-First make sure that the public key is present in your keychain:
-```
-gpg --import seedsigner_pubkey.gpg 
-```
-This command will import the public key, or return:
 ```
-key <...> not changed
+gpg --fetch-keys https://keybase.io/SeedSigner/pgp_keys.asc
 ```
+When the command completes successfully, it will show that this key was either imported or updated from Keybase.io. A numeric ID, as circled in red in the example below is known as the keys fingerprint. Please ignore the email address shown, because it is not part of any verification. 
+
+![SS - Keybase PubKey import with Fingerprint shown (New import or update of the key)](https://user-images.githubusercontent.com/91296549/174248861-7961c038-1fbf-47a1-a110-146cb218b1c8.jpg)  
+
+<details><summary>Learn more about how keybase.io helps you check that someone is who they say they are</summary>
+<p>
+  The Keybase.io website allows you to independently verify that the public key provided is authentic and that it belongs to the organization it claims to represent. 
+  Keybase has checked the pubkey cryptographically when it was saved in the 3 separate online locations. These are: on www.twitter.com/seedsigner, on the website www.seedsigner.com , and in the software repository at Github www.github.com/seedsigner.
+  You can verify those 3 separate Key locations yourself, by clicking the 3 blue badges on www.keybase.io/seedsigner. (The Twittter blue badge one is the most human-readble.) 
```

Comment:
"...by clicking on the links provided here or by clicking the three blue badges on..."?

My only minor concern is that Keybase potentially becomes a single point of failure; if we're absurdly paranoid, we should emphasize where we can cut Keybase out of the loop entirely.

Reply:
ok, its changed to: 
You can verify those three separate Key publications yourself, by clicking on any of the three blue badges at www.keybase.io/seedsigner. (The blue badge for the publication on Twitter is the most human-readable). 

Relying on Keybase: Yes, we can always remove keybase.io , and replace it with an ventual successor. (keyoxide.org is one such project being built out now, but is not yet live.)
So Keybase's purpose now, is mostly to host the pubkey, allow users to verify that this published pubkey matches the 3 other PubKey locations, and that they are still genuine, also not revoked. 
(To a lesser extent, to verify that the signing keys on Seedsigners (2) devices are still the same.). 

Yes, I will add language that Keybase can be removed/substituted, if it is compromised or creases to function.